lib/job-handlers.js

```diff
@@ -2,7 +2,7 @@ var _ = require('underscore');
 var JobManager = require('./job-manager');
 var ObserverManager = require('./observer-manager');
 var WebsocketEndpoint = require('./websocket-endpoint');
-var logger = require('log4js').getLogger('jobs-handler'); //jshint unused:false
+var logger = require('log4js').getLogger('jobs-handler'); // eslint-disable-line
```

Comment:
This probably means we should add some useful debug level logging rather than take the logger reference out. I filed https://github.com/juttle/outrigger/issues/105 to track this.